cmd/shell.go

```diff
@@ -30,6 +30,8 @@ import (
 	blog "github.com/letsencrypt/boulder/log"
 )
 
+var command = func() string { return path.Base(os.Args[0]) }()
```

Comment:
```suggestion
func command() string {
  return path.Base(os.Args[0])
}
```

The refactoring you did here kept it as a variable, but assigned that variable by creating a function literal, then immediately invoking it. It doesn't prevent changes. But declaring a function does prevent changes.

Reply:
Oh, derp, yep I should've realized this.